global.json

```diff
@@ -1,5 +1,5 @@
 {
     "sdk": {
-        "version": "5.0.301"
+        "version": "5.0.401"
```

Comment:
Why did you update this?

Reply:
I encountered an issue where I had a missing dotnet installation when I attempted to run dotnet restore, I tried to find 5.0.301 but it wasn't available for download anymore.

Reply:
5.0.7 from https://dotnet.microsoft.com/download/dotnet/5.0 seems to be the one.

Updating this is something we should probably do, but please don't include unrelated changes like this into PRs. Create a new PR for stuff like this.